sdk/lib/opentelemetry/sdk/configurator.rb

```diff
@@ -27,13 +27,17 @@ def initialize
         @text_injectors = nil
         @span_processors = []
         @use_mode = USE_MODE_UNSPECIFIED
-        @tracer_provider = Trace::TracerProvider.new
+        @resource = Resources::Resource.telemetry_sdk
       end
 
       def logger
         @logger ||= Logger.new(STDOUT)
       end
 
```

Comment:
How strict are we with rubocop on this project?  Asking for a friend.
```
lib/opentelemetry/sdk/configurator.rb:11:5: C: Metrics/ClassLength: Class has too many lines. [102/100]
```

Reply:
we are heavy users of `rubocop:disable`